.github/workflows/ci-cd.yml

```diff
@@ -57,70 +57,78 @@ jobs:
     needs: [eslint-backend, eslint-frontend, integration-tests, simulator-tests]
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-      with:
-        fetch-depth: '0'
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: "0"
+
+      - name: Push backend to Dockerhub
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: dagandersen/backend
+          username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+          password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+          workdir: backend
 
-    - name: Push backend to Dockerhub
-      uses: elgohr/Publish-Docker-Github-Action@master
-      with:
-        name: dagandersen/backend
-        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
-        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
-        workdir: backend
+      - name: Push frontend to Dockerhub
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: dagandersen/frontend
+          username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+          password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+          workdir: frontend
 
-    - name: Push frontend to Dockerhub
-      uses: elgohr/Publish-Docker-Github-Action@master
-      with:
-        name: dagandersen/frontend
-        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
-        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
-        workdir: frontend
+      - name: Push grafana to Dockerhub
```

Comment:
Why push grafana image to Dockerhub?

Reply:
Because we build a custom Grafana image where we bake in our configuration.